README.md

```diff
@@ -1,138 +1,49 @@
 # AutoGVP: Automated Germline Variant Pathogenicity
-Jung Kim, Ammar S. Naqvi, Ryan J. Corbett, Rebecca Kaufman, Zalman Vaksman, Miguel A. Brown,  Daniel P. Miller, Zhuangzhuang Geng, Phillip B. Storm, Adam C. Resnick, Jo Lynne Rokita, Douglas R. Stewart, Sharon J. Diskin
+Jung Kim, Ammar S. Naqvi, Ryan J. Corbett, Rebecca Kaufman, Zalman Vaksman, Miguel A. Brown,  Daniel P. Miller, Zhuangzhuang Geng, Phillip B. Storm, Adam C. Resnick, Douglas R. Stewart, Jo Lynne Rokita, Sharon J. Diskin
 
 ## AutoGVP Workflow  
 <img src="https://github.com/diskin-lab-chop/pathogenicity-assessment/blob/6848a369885fb9e03e75ec82afc97379457c0275/figures/germline-pathogenecity_flow.png" align="center" width = "600">
 
+For more detailed instructions, please visit the [user guide](https://github.com/diskin-lab-chop/AutoGVP/wiki/User-Guide) on our wiki.
+
 ## Clone the AutoGVP repository
 ```bash
 git clone https://github.com/diskin-lab-chop/AutoGVP.git
 ```
 
 ## Docker set-up
-
-### docker pull
+### docker pull, run, and execute
 ```bash
 docker pull pgc-images.sbgenomics.com/naqvia/autogvp:latest
 ```
 cd to your clone of `AutoGVP`
-
-### docker run
 Replace <CONTAINER_NAME> with any name and run the command below:
 ```
 docker run --platform linux/amd64 --name <CONTAINER_NAME> -d -v $PWD:/home/rstudio/AutoGVP pgc-images.sbgenomics.com/naqvia/autogvp:latest
-```
-### docker execute
-```bash
 docker exec -ti <CONTAINER_NAME> bash
 ```
-
 ### cd to AutoGVP directory within docker
 ```bash
 cd /home/rstudio/AutoGVP
 ```
 
-## How to Run
-
-### CAVATICA input ###
-1. Run the [Kids First Germline Annotation Workflow](https://github.com/kids-first/kf-germline-workflow/blob/v0.4.4/docs/GERMLINE_SNV_ANNOT_README.md) first.
-This workflow currently annotates variants with ClinVar (2022-05-07).
-2. Run the [Pathogenicity Preprocessing Workflow](https://github.com/d3b-center/D3b-Pathogenicity-Preprocessing), which performs ANNOVAR with InterVar and AutoPVS1 annotations.
-3. Download required db files, including variant submissions files (`variant_summary.txt` and `submission_summary.txt`) using download_db_files.sh (see below).
-4. Run `select-clinVar-submissions.R` within `scripts/` to consolidate ClinVar submission calls prior to running AutoGVP wrapper script. This step can be ran once and the `ClinVar-selected-submissions.tsv` file could be used for subsequent runs if clinVar versions remain the same. 
-5. Run AutoGVP wrapper script using `--workflow="cavatica"`
+## How to Run AutoGVP
 
 AutoGVP Requirements (recommended to place all in the `data/` folder):
 - VEP-, ANNOVAR-, and ClinVar annotated VCF file (`*VEP.vcf`)
```

Comment:
For custom run, it would be just VEP annotated VCF files
ANNOVAR annotation are done separately

Reply:
thanks, will update!